apps/web/locales/it.json

```
@@ -752,11 +752,15 @@
 		"settings": {
 			"BREADCRUMB": "[\"Pannello di controllo\", \"Impostazioni\"]",
 			"DANDER_ZONE": "Zona Pericolosa",
+			"working_hours": "Working hours",
 			"HEADING_DESCRIPTION": "Impostazione e gestione del tuo dashboard personale qui",
 			"ARE_YOU_SURE_TO_DELETE_USER": "Sei sicuro di voler eliminare questo utente?"
 		},
 		"settingsPersonal": {
 			"HEADING_TITLE": "Impostazioni Generali",
+			"goToTeamSettings": "Go to Team settings",
+			"WORKING_HOURS": "Orario di lavoro",
+			"goToTeamSettings": "Vai alle impostazioni del team",
```

Biome (2.3.14)

[error] The key goToTeamSettings was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)